### ISSUE TITLE
Sharing dataset?

### ISSUE THREAD

**Issue by gitlabspy** [RameenAbdal/StyleFlow#48]
Hi Rameen, so glad to see your work accepted by TOG!
I wonder if you could share your training dataset (w and corresponding labels)? Only for research and learning use.

**Comment by RameenAbdal** [comment]
Updated.

**Comment by gitlabspy** [comment]
You are the best! Thanks!

**Comment by gitlabspy** [comment]
@RameenAbdal There's about 8k labelled data, is it enough for training styleflow? I am trying to reproduce it in tensorflow :D
